module/VuFind/src/VuFind/Auth/OpenIDConnect.php

```diff
@@ -355,7 +355,7 @@ public function getSessionInitiator(string $target): ?string
             'client_id' => $this->getConfig('client_id'),
             'nonce' => $this->session->oidc_nonce,
             'state' => $this->session->oidc_state,
-            'scope' => 'openid profile email',
+            'scope' => $this->getConfig('scope'),
```

Comment:
Would it be worth having a hard-coded fallback default in case somebody runs this with an outdated .ini file?

```suggestion
            'scope' => $this->getConfig('scope') ?? 'openid profile email',
```

Reply:
Yes, of course. Thanks for pointing out!